bundles/org.openhab.binding.teslascope/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -17,9 +17,16 @@
 			<channel id="charging-state" typeId="charging-state"/>
 			<channel id="auto-conditioning" typeId="auto-conditioning"/>
 			<channel id="battery-range" typeId="battery-range"/>
+			<channel id="car-version" typeId="car-version"/>
```

Comment:
Is this really needed as a channel, or could it be published as a Thing property instead?

Reply:
Can rules access thing properties? It's sometimes helpful to trigger notifications based on a software version change.

Reply:
Yes. Example in JavaScript:
```javascript
rules.when()
    .thing('hue:bridge-api2:home')
    .changed().to('ONLINE')
    .then(event =>
    {
        var properties = things.getThing('hue:bridge-api2:home').rawThing.getProperties();
        var firmwareVersion = properties.get('firmwareVersion');
    })
    .build('Hue bridge online');
```

However, I don't think it's possible to trigger on property changes, for that you'd need a channel. But if the version can be considered relatively static, a cron expression or indirect trigger might be sufficient.

Reply:
I guess it depends what you consider static :) My Model 3 has had 45 Software updates in just under 2 years - an update on average every 16 days.

Reply:
I guess the Thing will come offline during a firmware upgrade, so you could trigger on the Thing status as shown in the example? Or would a cron expression be "good enough"?

Reply:
afaik teslascope won't report the car offline - so best would be a cron expression. Let me investigate how to update thing properties :)

Reply:
Fixed in latest commit

Reply:
i18n now regenerated